src/resolvers/Topics.js

```diff
@@ -1,21 +1,64 @@
 'use strict';
+const appInsights = require('applicationinsights');
+appInsights.setup(process.env.APPINSIGHTS_INSTRUMENTATIONKEY)
```

Comment:
We have this code repeated in quite a few places. Would it be possible to encapsulate this in one place to reduce the duplication (e.g. `src/loggingClient`)?

This has the added benefit that we can introduce a null-object ApplicationInsights client if the instrumentation key is not set so that we can enable running the app locally without requiring the instrumentation key to be specified which is a better developer experience.

Pseudo-code:

```js
const ikey = process.env.APPINSIGHTS_INSTRUMENTATIONKEY

let client;
if (ikey) {
  appInsights.setup(ikey);
  client = appInsights.getClient();
} else {
  client = {
    trackException: (ex) => console.log("exception " + ex),
    trackEvent: (route, details) => console.log("event " + route + " " + JSON.stringify(details))
  };
}

module.exports = {
  client: client
};
```